{"uid":"d7971162652a22a2","name":"Verify the account creation functionality.","fullName":"org.api.tests.AuthTest.createAccountTest","historyId":"9f332ada0a3dd0f3252d6838008d1fe5","time":{"start":1755279536186,"stop":1755279537902,"duration":1716},"description":"This test case verifies that a new user account can be created successfully using the SignUp API.","descriptionHtml":"<p>This test case verifies that a new user account can be created successfully using the SignUp API.</p>\n","status":"failed","statusMessage":"expected [User registered successfully!] but found [{\n    \"timestamp\": \"2025-08-15 17:38:57\",\n    \"status\": 500,\n    \"error\": \"System Error\",\n    \"message\": \"An unexpected error occurred\",\n    \"path\": \"uri=/api/auth/signup\",\n    \"solution\": \"Please try again later or contact support\",\n    \"errorCode\": \"SYS_001\"\n}]","statusTrace":"java.lang.AssertionError: expected [User registered successfully!] but found [{\n    \"timestamp\": \"2025-08-15 17:38:57\",\n    \"status\": 500,\n    \"error\": \"System Error\",\n    \"message\": \"An unexpected error occurred\",\n    \"path\": \"uri=/api/auth/signup\",\n    \"solution\": \"Please try again later or contact support\",\n    \"errorCode\": \"SYS_001\"\n}]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\n\tat org.testng.Assert.assertEquals(Assert.java:132)\n\tat org.testng.Assert.assertEquals(Assert.java:656)\n\tat org.testng.Assert.assertEquals(Assert.java:666)\n\tat org.api.tests.AuthTest.createAccountTest(AuthTest.java:59)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"setup","time":{"start":1755279535152,"stop":1755279536139,"duration":987},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"testStage":{"description":"This test case verifies that a new user account can be created successfully using the SignUp API.","status":"failed","statusMessage":"expected [User registered successfully!] but found [{\n    \"timestamp\": \"2025-08-15 17:38:57\",\n    \"status\": 500,\n    \"error\": \"System Error\",\n    \"message\": \"An unexpected error occurred\",\n    \"path\": \"uri=/api/auth/signup\",\n    \"solution\": \"Please try again later or contact support\",\n    \"errorCode\": \"SYS_001\"\n}]","statusTrace":"java.lang.AssertionError: expected [User registered successfully!] but found [{\n    \"timestamp\": \"2025-08-15 17:38:57\",\n    \"status\": 500,\n    \"error\": \"System Error\",\n    \"message\": \"An unexpected error occurred\",\n    \"path\": \"uri=/api/auth/signup\",\n    \"solution\": \"Please try again later or contact support\",\n    \"errorCode\": \"SYS_001\"\n}]\n\tat org.testng.Assert.fail(Assert.java:111)\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\n\tat org.testng.Assert.assertEquals(Assert.java:132)\n\tat org.testng.Assert.assertEquals(Assert.java:656)\n\tat org.testng.Assert.assertEquals(Assert.java:666)\n\tat org.api.tests.AuthTest.createAccountTest(AuthTest.java:59)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:324)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:74)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:123)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)\n","steps":[],"attachments":[{"uid":"2e8f8ec02e15a004","name":"Request","source":"2e8f8ec02e15a004.html","type":"text/html","size":1133},{"uid":"571af005a05c6031","name":"HTTP/1.1 500 ","source":"571af005a05c6031.html","type":"text/html","size":1402}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":2,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"org.api.tests.AuthTest"},{"name":"testClass","value":"org.api.tests.AuthTest"},{"name":"testMethod","value":"createAccountTest"},{"name":"parentSuite","value":"API Test Suite"},{"name":"suite","value":"API Test"},{"name":"subSuite","value":"org.api.tests.AuthTest"},{"name":"host","value":"pkrvmubgrv54qmi"},{"name":"thread","value":"2430@pkrvmubgrv54qmi.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Create Account"},{"name":"epic","value":"Authentication"},{"name":"feature","value":"Auth Service"},{"name":"severity","value":"blocker"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"blocker","retries":[{"uid":"9b0824cedbf87f5","status":"unknown","time":{"start":1755236737955,"stop":1755236737956,"duration":1}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":0,"skipped":0,"passed":0,"unknown":3,"total":5},"items":[{"uid":"88b1b5c549ff503f","reportUrl":"https://rangegowdaym.github.io/apiassured/10//#testresult/88b1b5c549ff503f","status":"failed","statusDetails":"expected [User registered successfully!] but found [{\n    \"timestamp\": \"2025-08-15 15:55:42\",\n    \"status\": 500,\n    \"error\": \"System Error\",\n    \"message\": \"An unexpected error occurred\",\n    \"path\": \"uri=/api/auth/signup\",\n    \"solution\": \"Please try again later or contact support\",\n    \"errorCode\": \"SYS_001\"\n}]","time":{"start":1755273341365,"stop":1755273342881,"duration":1516}},{"uid":"b10d9a80ac9e8871","reportUrl":"https://rangegowdaym.github.io/apiassured/9//#testresult/b10d9a80ac9e8871","status":"unknown","time":{"start":1755270222343,"stop":1755270222343,"duration":0}},{"uid":"5184726adbd99f8a","reportUrl":"https://rangegowdaym.github.io/apiassured/8//#testresult/5184726adbd99f8a","status":"unknown","time":{"start":1755269375348,"stop":1755269375348,"duration":0}},{"uid":"5e60b12318d39bbc","reportUrl":"https://rangegowdaym.github.io/apiassured/7//#testresult/5e60b12318d39bbc","status":"unknown","time":{"start":1755267169534,"stop":1755267169534,"duration":0}}]},"tags":[]},"source":"d7971162652a22a2.json","parameterValues":[]}